app/lib/actions/trigger_proxy_batch.rb

```diff
@@ -40,7 +40,8 @@ def trigger_remote_tasks_batch
         output[:planned_count] += group.size
       end
     rescue => e
-      action_logger.warn "Could not trigger task on the smart proxy: #{e.message}"
+      action_logger.warn "Could not trigger task on the smart proxy"
+      action_logger.warn e
```

Comment:
Won't this be just the message without backtrace?

Reply:
Luckily not

```
2022-02-08T07:46:11 [W|bac|113dbd4d] Could not trigger task on the smart proxy
2022-02-08T07:46:11 [W|bac|113dbd4d] undefined method `[]' for nil:NilClass (NoMethodError)
 113dbd4d | /opt/theforeman/tfm/root/usr/share/gems/gems/foreman-tasks-5.2.0/app/models/foreman_tasks/remote_task.rb:42:in `update_from_batch_trigger'
 113dbd4d | /opt/theforeman/tfm/root/usr/share/gems/gems/foreman-tasks-5.2.0/app/models/foreman_tasks/remote_task.rb:36:in `block (2 levels) in batch_trigger'
-----B<-----SNIP-----B<-----
```